### PR TITLE
Adding ABOUT and DOCUMENTATION markdown files for the website.

### PR DESCRIPTION
**Pull request by greglucas**
Creating a docs folder with ABOUT and DOCUMENTATION markdown files that the main website can display.

### CODE REVIEW

File: docs/DOCUMENTATION.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+"speed": 7750,
+"composition": {"O": 1e11,
+                "O2": 1e6,
+                "N2": 1e6,
+                "He": 1e6,
+                "H": 1e4},
+"accommodationModel": "SESAM",
+"accommodationParameters": {"alpha": 0,
+                            "ms": 0}
+}
```
Comment (jennyknuth) [inline docs/DOCUMENTATION.md]: For this to show up nicely formatted on the other side, I think you would need to contain it in tick marks for code, and ideally indicate what language it is, e.g.,  ```json… but this is plenty good for now. Ditto for the output.

File: docs/DOCUMENTATION.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+forceCoefficent: float
+
+    The force coefficient of the object
+
+### Example Output Payload
+
+{
+"dragCoefficient": 2.8,
+"projectedArea": 3,
+"forceCoefficient": 1.2
```
Comment (jennyknuth) [inline docs/DOCUMENTATION.md]: I believe Marcin wanted us to add 'energyAccommodation' as an output as well?

File: docs/DOCUMENTATION.md (hunk 1 above)
Comment (greglucas) [reply docs/DOCUMENTATION.md]: Thanks. Just updated it.

File: docs/DOCUMENTATION.md (hunk 2 above)
Comment (greglucas) [reply docs/DOCUMENTATION.md]: Added that in as well.

**Pull request merged**